tests/benchdnn/inputs/conv/test_conv_regression

```diff
@@ -15,3 +15,5 @@
 
 #
 --batch=harness_conv_regression_general
+
+--reset --dt=bf16 --bia-dt=bf16 --dir=FWD_I g4mb2_ic4oc4_ih6oh4kh3sh1dh0ph0_iw6ow4kw3sw1dw0pw0_n"aarch64_jit_dw_bia_dt_bf16"
```

Comment:
I'm sorry I gave a wrong pointer in the beginning. Could you, please, put it under `harness_conv_regression_general`, it is the proper list to collect those single line cases for regressions.